### PR TITLE
mail.google.com - Scrollbar actualy visible

### LINKED ISSUE #541: [Dynamic] Scrollbars in gmail are virtually invisible
image

### PR DESCRIPTION
**Pull request by Gusted**
resolves #541

The left side of the corner is dark? I can't find any CSS property that cause this or to fix this or I'm seeing something over the head.

# Before

# After

### CODE REVIEW

**Comment by Gusted** [comment]
Second opnion from someone for this

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -1539,6 +1539,11 @@ img.Ha
 .gb_df
 img[src$="google_gsuite"]
 
+CSS
+::-webkit-scrollbar-thumb {
+    background-color: #424242 !important;
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config:1544]: Please check how it looks in Light mode.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:1544]: Very weird it doesn't show the black bar asides I think some CSS property is setted that cause this.

**Pull request closed without merging**